docs/README.md

```diff
@@ -1,12 +1,27 @@
+[La version française suit.](#---------------------------------------------------------------------)
```

Comment:
Shouldn't think link to `#Documents sur track-web` instead of the dividing line?